src/core/localization/translations/en/common.json

```diff
@@ -2,13 +2,13 @@
   "km": "km",
   "m": "m",
   "to": "to",
-  "logout": "Logout",
+  "logout": "Log out",
   "save": "Save",
   "cancel": "Cancel",
   "create": "Create",
   "disasters": "Disasters",
   "loading": "Loading...",
-  "loading_events": "Loading events",
+  "loading_events": "Loading disasters",
```

Comment:
@palina-krukovich review this change please.

Reply:
looks good